queries/editorconfig/highlights.scm

```diff
@@ -0,0 +1,51 @@
+(comment) @comment @spell
+
+; consistency with toml and ini
+(section
+  (section_name) @type)
+
+(character_choice
+  (character) @constant)
+
+(character_range
+  start: (character) @constant
+  end: (character) @constant)
+
+[
+  "["
+  "]"
+  "{"
+  "}"
+] @punctuation.bracket
+
+[
+  ","
+  (path_separator)
+] @punctuation.delimiter
+
+[
+  "-"
+  "="
+  ".."
+  (negation)
+  (wildcard_characters)
+  (wildcard_any_characters)
+  (wildcard_single_character)
+] @operator
+
+(escaped_character) @string.escape
+
+(pair
+  key: (identifier) @property)
```

Comment:
Values should be highlighted as `@string` by default.

Reply:
Why? That's not how the legacy syntax works, and I don't see any documentation supporting that behavior.

Reply:
I at least get this highlighting when opening a `.editorconfig` file with `nvim --clean`, numbers *are* highlighted as numbers but not any other special value (space, true, lf).

![image](https://github.com/user-attachments/assets/1c3067a5-240d-4a5d-9463-77fff7e0a812)

Reply:
Yeah, I was thinking of numbers. We _could_ distinguish (override) these with a `#lua-match?`.

Reply:
But it's a question of semantics: if the format treats every value -- regardless of content -- as a literal, `@string` is arguably correct. But then it's not a question of "default"; _everything_ must be a string. I lack the domain knowledge here to decide. @gpanders ?

Reply:
In any case, since the parser exposes them, we should capture them.